docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -728,8 +728,8 @@ Each time deployment is requested, a container image build will be implicitly tr
 
 === Deploying
 
-When deployment is enabled, the kubernetes extension will selected the resources specified by `quarkus.kubernetes.deployment.target` and deploy them.
-This assumes that a `.kube/config` is available in your user directory that points to the target kubernetes cluster.
+When deployment is enabled, the Kubernetes extension will selected the resources specified by `quarkus.kubernetes.deployment.target` and deploy them.
```

Comment:
```suggestion
When deployment is enabled, the Kubernetes extension will select the resources specified by `quarkus.kubernetes.deployment.target` and deploy them.
```

Reply:
Not a fault of this PR of course, but while you are at it, let's fix this typo :)

Reply:
Done.